examples/specs/line_quarter.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Stock price mean per quarter broken down by years.",
+  "data": {"url": "data/stocks.csv", "formatType":"csv"},
+  "mark": "point",
```

Comment:
@bobocandys  @domoritz   not sure if you want this to be line or point.   In any case, either rename the file name or the change the mark to make it consistent.  ;)